indico/indico.conf

```diff
@@ -34,8 +35,8 @@ DEFAULT_LOCALE = os.environ.get('INDICO_DEFAULT_LOCALE') or 'en_GB'
 
 CELERY_BROKER = os.environ['CELERY_BROKER']
 
-STORAGE_BACKENDS = {'default': 'fs:/opt/indico/archive'}
-ATTACHMENT_STORAGE = 'default'
+STORAGE_BACKENDS = json.loads(os.environ['INDICO_STORAGE_DICT']) or {'default': 'fs:/opt/indico/archive'}
```

Comment:
this will fail if the env var is not defined! might want to use `.get()`. also, `json.loads()` on `None` or an empty string will fail as well